### DISCUSSION TITLE
Support for configurable auto dark mode in GUI

### DISCUSSION THREAD

**Discussion by agneevX** [jesec/flood#290] [Ideas]
Currently, the GUI defaults to system auto dark mode.

However, this functionality is disabled if the theme toggle is set to light or dark. 
The only way to restore the system preference is to remove the bool flag manually from browser storage.

What I propose is three modes -- System, dark, light.

**Comment by jesec** [comment]
Good idea. Though, design-wise, it would be challenging to implement a three-way toggle.

**Reply by agneevX** [reply]
I really like how [Homer](https://github.com/bastienwirtz/homer) does it:

https://user-images.githubusercontent.com/19761269/117549297-32736a00-b057-11eb-8ce8-da968e8f3235.mov
